other/material/import_material_from_jarvis_db_entry.ipynb

```diff
@@ -1,137 +1,749 @@
 {
```

Comment:
<p>Was there also a part where we extracted the properties, by chance?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>